float32[] location
float32[] speed
float32[] load
float32[] voltage
float32[] temperature
float32[] current
uint8[] error
